stan/math/prim/arr/fun/promote_elements.hpp

```diff
@@ -3,6 +3,7 @@
 
 #include <stan/math/prim/scal/fun/promote_elements.hpp>
 #include <vector>
+#include <cstddef>
```

Comment:
Why is cstddef being added?  It looks like it would have passed header tests before.